sdk/client/src/blocking.rs

```diff
@@ -26,6 +26,8 @@ use serde::{de::DeserializeOwned, Serialize};
 
 const REQUEST_TIMEOUT: u64 = 10_000;
 
+const USER_AGENT: &'static str = concat!("diem-client-sdk-rust / ", env!("CARGO_PKG_VERSION"));
```

Comment:
The compiler should already be able to infer the appropriate lifetime for this so you should be able to have the type be `&str` instead of `&'static str`